test/helpers/analysis.spec.ts

```diff
@@ -1,15 +1,18 @@
 import { resolve } from 'pathe'
 
 import { extractConfigFromFile } from '../../packages/runtime/src/helpers/analysis'
+import logger from '../../packages/runtime/src/helpers/logger'
 
 describe('static source analysis', () => {
   beforeEach(() => {
-    //  Spy on console.error
-    jest.spyOn(console, 'error').mockImplementation(() => {})
+    //  Spy on logger.error
+    // TODO: Request for PR comments - Do we need to mock this?
+    // We can just leave it as is, but it will print out the error message
+    jest.spyOn(logger, 'error')
```

Comment:
I wasn't sure what to do here.

`logger.error` expects a return type of a Winston `Logger`. We don't really need to mock this unless we care about the test output including the `logger.error` strings.

Thoughts?

Reply:
Updated to mock it and return the logger so we can chain if we needed to. Thanks, @orinokai for the help.